packages/grafana-ui/src/components/Table/TableNG/styles.ts

```diff
@@ -206,3 +225,6 @@ export const getTooltipStyles = (theme: GrafanaTheme2, textAlign: TextAlign) =>
     },
   }),
 });
+
+export const getActiveCellSelector = (isNested?: boolean) =>
```

Comment:
I wish we didn't need something like this. The fact is, though, that sometimes you'll want to target hover and selection and you will be within a nested element, and sometimes you will want to target it and you'll be on the rdg-cell itself, so this utility is just meant to help wrap that complexity up a bit.